### PR TITLE
DEVDOCS-4824 + DEVDOCS-3526: [migrate] Many, remove references to Stoplight + unused docs

### PR DESCRIPTION
**Pull request by stanzikratelbc**
[DEVDOCS-4824]: removed references to Stoplight in multiple documents

[DEVDOCS-4824]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4824?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,20 +2,20 @@
...
 
-The following is a set of guidelines for contributing to BigCommerce Developer Documentation. These are just guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+The following is a set of guidelines for contributing to BigCommerce's Developer Documentation. These are guidelines, not rules. Use your best judgment, and make proposed changes to this document in a pull request.
 
 ## Making a Quick Edit
+
 The easiest way to edit a file is using GitHub's web interface:
 
-1. Navigate to the file in github ([making-requests.md](/bigcommerce/dev-docs/blob/master/docs/api-docs/getting-started/making-requests.md), for example)
+1. Navigate to the file in github ([making-requests.md](NEED NEW URL), for example)
```
Comment (bc-tgomez) [inline CONTRIBUTING.md:18]: Line 1 needs the new URL here.

**Review by bc-tgomez — DISMISSED** [review]
Great edits! Some of the URLs are out of date and need replacing. **README.MD** has outdated URLs in Line 3. Also, I added a commit for minor linting issues.

File: README.md [hunk 2]
```diff
@@ -25,19 +25,19 @@ Other repositories that contain markdown files and static assets that appear on
 ├── .gitignore                         # gitignore
 ├── .nojekyll                          # 
 ├── .spelling                          # mdspell dictionary
-├── .stoplight.json                    # Stoplight Platform edit view configuration file
 ├── .textlintrc                        # textlint configuration file
 ├── CONTRIBUTING.md                    # guidelines for contribution
 ├── Gruntfile.js                       # grunt configuration file
 ├── package-lock.json                  # packages for grunt, linting, and spell checking
 ├── package.json                       # packages for grunt, linting, and spell checking
 ├── pull_request_template_CODEOWNER.md # template for codeowner dev merges
```
Comment (slsriehl) [inline README.md:33]:
```suggestion
```

Comment (slsriehl) [reply README.md:33]: codeowner template is elsewhere

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,64 +2,57 @@
...
+- Capitalize the subject line.
+- Do not end the subject line with a period.
+- Use the body to explain what and why versus how.
 
 ## Style Guides
-* [Documentation Style Guide](_project/_doc_style_guide.md)
-* [API Specification Style Guide](_project/_spec_style_guide.md)
+
+- [Documentation Style Guide](_project/_doc_style_guide.md)
+- [API Specification Style Guide](_project/_spec_style_guide.md)
```
Comment (slsriehl) [inline CONTRIBUTING.md:55]: (this comment is scope sprawl & should be used to make another ticket) --

I don't think we have this `_project` directory anymore, and the style guides almost certainly aren't current

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,64 +2,57 @@
...
 ## Editing Locally
-For more complex changes, fork and edit locally: 
+
+For more complex changes, fork and edit locally:
 
 1. Fork `bigcommerce/dev-docs`
 2. `git clone` the fork to your local machine
-3. Make edits in your favorite code editor on a new branch
-4. Commit and pushes changes to your remote repo
+3. Make edits in your IDE of choice on a new branch
```
Comment (slsriehl) [inline CONTRIBUTING.md:37]:
```suggestion
3. Make edits on a new branch in your IDE of choice.

```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -2,64 +2,57 @@
...
+10. Click **Create pull request**.
 
 That's it! You're done.
 
 ## Editing Locally
-For more complex changes, fork and edit locally: 
+
+For more complex changes, fork and edit locally:
 
 1. Fork `bigcommerce/dev-docs`
```
Comment (slsriehl) [inline CONTRIBUTING.md:35]:
```suggestion
1. Fork `bigcommerce/dev-docs`.

```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -2,64 +2,57 @@
...
 2. `git clone` the fork to your local machine
```
Comment (slsriehl) [inline CONTRIBUTING.md:36]:
```suggestion
2. `git clone` the fork to your local machine.

```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -2,64 +2,57 @@
...
-3. Make edits in your favorite code editor on a new branch
-4. Commit and pushes changes to your remote repo
+3. Make edits in your IDE of choice on a new branch
+4. Commit and push changes to your remote repo
 5. Create a [pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) against `bigcommerce/dev-docs/master`
```
Comment (slsriehl) [inline CONTRIBUTING.md:39]:
```suggestion
5. Create a [pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) to `bigcommerce:dev-docs/main`.

```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -2,64 +2,57 @@
...
-2. Click the **pencil** icon to **Edit This File**
-3. Make the edit
-4. Type a commit message
-5. Select **Create a new branch for this commit and start a pull request**
-6. Give the branch a descriptive name
-7. Click **Propose file change**
-8. Give the pull request a descriptive title
-9. Fill in the pull request description
-10. Click **Create pull request**
+1. Navigate to the file in github ([making-requests.md](https://github.com/bigcommerce/dev-docs/blob/main/docs/api-docs/getting-started/making-requests.mdx), for example).
```
Comment (slsriehl) [inline CONTRIBUTING.md:18]:
```suggestion
1. Navigate to the file in GitHub. For example, [making-requests.md](https://github.com/bigcommerce/dev-docs/blob/main/docs/api-docs/getting-started/making-requests.mdx).

```

File: docs/api-docs/headless/customers.mdx [hunk 9]
```diff
@@ -59,10 +59,6 @@ To address this need, BigCommerce provides the [Current Customers API](/api-docs
 
 ## Surfacing customer group pricing
 
-When querying the GraphQL Storefront API, customer-specific pricing will be reflected in query results if the customer is first logged in using the [customer login mutation](#) or a [customer impersonation token](/api-docs/storefront/graphql/graphql-storefront-api-overview#customer-impersonation-tokens).
```
Comment (slsriehl) [inline docs/api-docs/headless/customers.mdx:62]: why are we taking this first sentence out? it seems unrelated to the PR.

File: docs/partner-apps/bundleb2b/b2b-edition.mdx [hunk 10]
```diff
@@ -151,12 +151,10 @@ For a full list of available modules, see the [BundleB2B Developer Guide](https:
 
 You can find instructions on how to get a B3 authentication token on the [BundleB2B Developer Guide](https://developer.bundleb2b.net/storefront/api-call.html) page.
 
-For the complete list of B3 API endpoints, see the [BundleB2B OpenAPI Documentation](https://bundleb2b.stoplight.io/).
```
Comment (slsriehl) [inline docs/partner-apps/bundleb2b/b2b-edition.mdx:154]: this should stay. As of now, Bundle B2B still uses Stoplight.

File: docs/partner-apps/bundleb2b/b2b-edition.mdx [hunk 11]
```diff
@@ -151,12 +151,10 @@ For a full list of available modules, see the [BundleB2B Developer Guide](https:
 
 You can find instructions on how to get a B3 authentication token on the [BundleB2B Developer Guide](https://developer.bundleb2b.net/storefront/api-call.html) page.
```
Comment (slsriehl) [inline docs/partner-apps/bundleb2b/b2b-edition.mdx:153]:
```suggestion

For the complete list of B3 API endpoints, see the [BundleB2B OpenAPI Documentation](https://bundleb2b.stoplight.io/).
```

File: docs/partner-apps/bundleb2b/b2b-edition.mdx [hunk 12]
```diff
@@ -151,12 +151,10 @@ For a full list of available modules, see the [BundleB2B Developer Guide](https:
...
 
-For the complete list of B3 API endpoints, see the [BundleB2B OpenAPI Documentation](https://bundleb2b.stoplight.io/).
 
 ## Additional BundleB2B resources
 
 - [Quick Start](https://developer.bundleb2b.net/storefront/quick-start.html)
```
Comment (slsriehl) [inline docs/partner-apps/bundleb2b/b2b-edition.mdx:157]:
```suggestion
- [Quick Start](https://developer.bundleb2b.net/storefront/quick-start.html)
- [BundleB2B API Reference](https://bundleb2b.stoplight.io/)
```

**Review by slsriehl — CHANGES_REQUESTED** [review]
The main thing is that BundleB2B still uses Stoplight for their docs, so I made suggestions to leave theirs in.

I also made suggestions to add periods to a list of steps, like the other lists here.

Finally, the style guide recommends that we avoid parentheticals, so i've suggested that here.

File: CONTRIBUTING.md (hunk 1 above)
Comment (slsriehl) [reply CONTRIBUTING.md]: added.

File: docs/partner-apps/bundleb2b/b2b-edition.mdx [hunk 13]
```diff
@@ -151,12 +151,10 @@ For a full list of available modules, see the [BundleB2B Developer Guide](https:
...
-- [BundleB2B API Reference](https://bundleb2b.stoplight.io/)
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/partner-apps/bundleb2b/b2b-edition.mdx]: overwritten by B2B article changes.

File: README.md [hunk 14]
```diff
@@ -1,43 +1,41 @@
 # BigCommerce Developer Documentation
 
-This repository contains the markdown files and static assets powering [developer.bigcommerce.com](https://developer.bigcommerce.com/), which includes [API Docs](https://developer.bigcommerce.com/docs), [Theme Docs](https://developer.bigcommerce.com/stencil-docs), [BigCommerce for WordPress](https://developer.bigcommerce.com/bigcommerce-for-wordpress), and [Legacy Docs](https://developer.bigcommerce.com/legacy).
+This repository contains the markdown files and static assets powering [developer.bigcommerce.com](https://developer.bigcommerce.com/), which includes [API Docs](https://developer.bigcommerce.com/docs), [Theme Docs](https://developer.bigcommerce.com/stencil-docs), and [BigCommerce for WordPress](https://developer.bigcommerce.com/bigcommerce-for-wordpress).
```
Comment (slsriehl) [inline README.md:3]:
```suggestion
This repository contains the markdown files and static assets powering [developer.bigcommerce.com](https://developer.bigcommerce.com/), which includes [API Docs](https://developer.bigcommerce.com/docs), [Theme Docs](https://developer.bigcommerce.com/stencil-docs), and [BigCommerce for WordPress](https://developer.bigcommerce.com/bigcommerce-for-wordpress/overview).

```

**Review by slsriehl — APPROVED** [review]

**Pull request merged**